pkg/addon/addon.go

```diff
@@ -599,10 +599,18 @@ func getClusters(args map[string]interface{}) []string {
 		return nil
 	}
 	cc, ok := ccr.([]string)
+	if ok {
+		return cc
+	}
+	ccrslice, ok := ccr.([]interface{})
 	if !ok {
 		return nil
 	}
-	return cc
+	ccstring := make([]string, len(ccrslice))
+	for i := range ccrslice {
+		ccstring[i] = ccrslice[i].(string)
```

Comment:
Although it may not happen, we should skip the items that cannot be converted to string, just in case.